src/Contract/Operation/Entropyable.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace loophp\collection\Contract\Operation;
+
+use loophp\collection\Contract\Collection;
+
+/**
+ * @template TKey
+ * @template T
+ */
+interface Entropyable
+{
+    /**
+     * Calculate the normalized Shannon entropy for each collection items.
+     *
+     * If you're looking for one single result, you must get the last item using
+     * `last` operation.
```

Comment:
let's make the doc block description here match the api docs descriptions like the other operations